libs/components/src/lib/about-me/index.tsx

```diff
@@ -1,24 +1,37 @@
 import { FC } from "react";
-import cls from "classnames";
 
-import "../tailwind-imports.css";
+import { TextArea } from "../TextArea";
+import { TextField } from "../TextField";
+import { useData } from "./index.hook";
+import classes from "./index.module.scss";
 
 /* eslint-disable-next-line */
 export interface AboutMeProps {}
 
 export const AboutMe: FC<AboutMeProps> = () => {
-  return (
-    <div className={cls(
-      "p-1",
-      "rounded-lg",
-      "border",
-      "text-slate-400",
-      "bg-black"
-    )}
-    >
-      <h1>Welcome to AboutMe!</h1>
-    </div>
-  );
-}
+  const data = useData();
+  const renderBody = () => {
+    return (
+      <>
+        <TextField
```

Comment:
if you want to use shallow rendering it's better to use it for Header and TextArea separately.
```
renderHear()
renderTextArea()
```

Reply:
okay, it's done.